src/style.css

```diff
@@ -1,3 +1,80 @@
-body {
+html {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+* {
   margin: 0;
 }
+
+li {
+  list-style: none;
+  margin-left: 20px;
+}
+
+li:first-child {
+  margin-left: 0;
+}
+
+ul {
+  padding: 0;
+  display: flex;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.header {
+  position: relative;
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+}
+
+.header-top-logo {
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  display: flex;
+}
+
+.active:hover {
+  color: #00acdc;
+}
+
+.is-active {
```

Comment:
The '.is-active' class is used to style active links. Ensure that this class is consistently applied in the HTML to avoid styling inconsistencies.